charts/steward/README.md

```diff
@@ -156,10 +156,50 @@ Common parameters:
 | <code>pipelineRuns.<wbr/>resourceQuota</code> | (string)<br/> The resource quota to be created in every pipeline run namespace. The value must be a string containing a complete `resourcequotas` resource manifest in YAML format. The `.metadata` section of the manifest can be omitted, as it will be replaced anyway. See the [Kubernetes documentation of resource quotas][k8s-resourcequotas] for details about Kubernetes resource quotas.| none |
 
+### Feature Flags
+
+| Parameter | Description | Default |
+|---|---|---|
+| <code>featureFlags</code> | (string)<br/> Feature flag definition.  | empty |
+
+The feature flags definition is a string containing any number of feature flag names separated by any non-empty sequence of comma (`,`) and/or whitespace (space, horizontal tab, vertical tab, carriage return, newline, form feed). Separators at the beginning and the end of the string are allowed.
+
+A feature flag gets _enabled_ if the name is either prefixed with a plus sign (`+`) or not disabled.
+A feature flag gets _disabled_ if the name is prefixed with a minus sign (`-`).
+
+**Example 1: Typical format**
+
+    Flag1 +Flag2 -Flag3
+
+or alternatively using comma and space as separator:
+
+    Flag1, +Flag2, -Flag3
+
+or alternatively using newline as separator:
+
+    Flag1
+    +Flag2
+    -Flag3
+
+`Flag1` and `Flag2` get enabled, while `Flag3` gets disabled.
+
+**Example 2: Untypical but correct format**
+
+    , Flag1 +Flag2, ,,,,
+    Flag3,
+
+It has the same effect as in example 1. The definition string has leading and trailing separators, uses different separator sequences.
+
+| Feature Flag | Description | Default |
+| --- | --- | --- |
+| RetryOnInvalidPipelineRunsConfig | If enabled, the pipeline run controller retries reconciling PipelineRun objects in case the controller configuration (in ConfigMaps) is invalid or cannot be loaded. It is assumed that the condition can be detected by a monitoring tool, triggers an alert and operators fix the issue in a timely manner. By that operator errors do not immediately break user pipeline runs. However, processing of PipelineRun objects may be delayed significantly in case of invalid configuration.<br/><br/> If disabled, the current behavior is used: immediately set all unfinished PipelineRun objects to finished with result code `error_infra`.<br/><br/>  The new behavior is supposed to become the default in a future release of Steward. | disabled |
```

Comment:
```suggestion

#### List of Defined Feature Flags

| Feature Flag | Description | Default |
| --- | --- | --- |
| `RetryOnInvalidPipelineRunsConfig` | If enabled, the pipeline run controller retries reconciling PipelineRun objects in case the controller configuration (in ConfigMaps) is invalid or cannot be loaded. It is assumed that the condition can be detected by a monitoring tool, triggers an alert and operators fix the issue in a timely manner. By that operator errors do not immediately break user pipeline runs. However, processing of PipelineRun objects may be delayed significantly in case of invalid configuration.<br/><br/> If disabled, the current behavior is used: immediately set all unfinished PipelineRun objects to finished with result code `error_infra`.<br/><br/>  The new behavior is supposed to become the default in a future release of Steward. | disabled |
```